runtime/doc/if_lua.txt

```diff
@@ -31,7 +34,8 @@ current values of `package.path` or `package.cpath`. If you happened to delete
 some paths from there you need to reset 'runtimepath' to make them readded.
 
 Note 3: paths from 'runtimepath' which contain semicolons cannot be put into 
-`package.[c]path` and thus are ignored.
+`package.[c]path` for that being a semicolon-separated list and thus are 
+ignored.
```

Comment:
Perhaps:

paths from 'runtimepath' which contain semicolons are not added to `package.[c]path`, because those use the semicolon as a path separator.